pyswarms/utils/decorators/decorators.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+
+def cost(cost_func):
+    """A decorator to mark the cost function
+
+    This decorator provides a way to simplify the cost function. The cost function
+    needs to resemble a strategy that can be applied to every particle in the swarm
+    to assign a cost. This means that the decorated cost function must use an array
+    of shape :code:`(dimensions, )` as an argument and return a :code:`float` or an
+    :code:`int`.
+    
+    Parameters
+    ----------
+    
+    cost_func : function
+        A function that can be used as cost function in the optimization (must return a :code:`float` or an :code:`int`)
```

Comment:
This line's a bit long (?) I don't think black formats docstrings, can `flake8` catch this line?  What does `flake8 decorators.py` say?